conan/tools/google/bazeldeps.py

```diff
@@ -60,7 +60,7 @@ def _get_requirements(conanfile, build_context_activated):
     """
     # All the requirements
     host_req = conanfile.dependencies.host
-    build_req = conanfile.dependencies.direct_build  # tool_requires
+    build_req = conanfile.dependencies.build  # tool_requires
```

Comment:
@memsharded Without this, we can not get the transitive dependencies from any build-context requirement so the test would fail. Apart from that, PkgConfigDeps was using it but CMakeDeps and this one were not.